README.md

```diff
@@ -2,391 +1,312 @@
+For example, the SDK helps with creating a signed request for Learnosity:
 
-3.) Download the source code and integrate into your project. This requires the external libraries which can be found in learnosity-sdk-java/Dist/Light/RequiredJars
+![image-signed-request-creation.png](docs/images/image-signed-request-creation.png)
 
-4.) To use as a dependency in a maven project see below 'Using with a file based repository'
+Once the SDK has created the signed request for you, your app sends that on to an API in the Learnosity cloud, which then retrieves the assessment you are asking for, as seen in the diagram below:
 
-## Test
+![image-assessment-retrieval.png](docs/images/image-assessment-retrieval.png)
 
-In order to check that you've added the required code correctly, you can download src/LearnositySdk/Test and try to run the test file. If it runs successfully, all should be fine.
+This scenario is what you can see running in the Quick start guide example ([see below](#quick-start-guide)).
 
-The Test.java class also gives some examples on how to use the sdk.
+There's more features, besides. See the detailed list of SDK features on the [reference page](REFERENCE.md).
 
-## Usage
+[(Back to top)](#table-of-contents)
 
-### Init
+## Requirements
 
-The Init class is used to create the necessary *security* and *request* details used to integrate with a Learnosity API. Most often this will be a JavaScript object.
 
-The Init constructor takes either 3 arguments:
+1. Runtime libraries for Java installed. We've used Eclipse Adoptium/Temurin 11 LTS (previously known as 'AdoptOpenJDK'). ([instructions](https://projects.eclipse.org/projects/adoptium.temurin/downloads))
 
- * String  service type
- * Object  security details (**no secret**)
- * String  secret
+2. Maven installed ([instructions](https://maven.apache.org/download.cgi)). We tested on Maven 3.8.4.
 
-or 4 arguments:
+3. Docker installed (required for the example tutorial on this page). ([instructions](https://www.docker.com/get-started)). We tested on Docker version 3.x.
 
- * String  service type
- * Object  security details (**no secret**)
- * String  secret
- * Object  request details
+Note: just looking for a .WAR file? Look in the Maven target directory.
 
-The security and request Object can be of any type that can be parsed into a org.json.JSONObject. Examples are org.json.JSONOBject, a valid JSON String or a map. (See Test.java for examples). Learnosity recommends that you pass the request details as a JSONObject, or a String generated from a JSONObject to the constructor. This way, issues arising from adding/removing whitespace can be avoided.
+Not using Java? See the [SDKs for other languages](https://help.learnosity.com/hc/en-us/sections/360000194318-Server-side-development-SDKs).
 
-If you have to set an action attribute (for calls to data api, if not using the DatApi class), you can use the setAction method of class Init.
+### **Supported Java Versions**
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
### **Supported Java Versions**

```